numpy/core/src/npymath/ieee754.c.src

```diff
@@ -759,13 +780,18 @@ int npy_clear_floatstatus(void)
 
 #else
 
-int npy_get_floatstatus(void)
+int npy_get_floatstatus(void *param)
```

Comment:
Why pass as `void *` instead of `char *`?

Reply:
no particular reason. This way any pointer argument will suffice and silently cast, if it is `char*` I will need to cast everywhere it is called, no?